api/http/proxy/manager.go

```diff
@@ -45,7 +46,11 @@ func (manager *Manager) CreateAndRegisterProxy(endpoint *portainer.Endpoint) (ht
 		} else {
 			proxy = manager.proxyFactory.newHTTPProxy(endpointURL)
 		}
-	} else {
+	}
+	case "npipe":
+		var namedPipePath = strings.Replace(endpoint.URL, "npipe:", "", 1)
```

Comment:
I think that you can use `endpoint.URL.Path` here instead of `strings.Replace(endpoint.URL, "npipe:", "", 1)`